packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3472,23 +3478,145 @@ const TextStyle _messageStyle = TextStyle(
   height: 1.2,
 );
 
-class _ExitWidgetSelectionButtonWrapper extends StatefulWidget {
-  const _ExitWidgetSelectionButtonWrapper({
-    required this.button,
-    required this.buttonKey,
+class _ExitWidgetSelectionButtonGroup extends StatefulWidget {
+  const _ExitWidgetSelectionButtonGroup({
+    required this.exitWidgetSelectionButtonBuilder,
+    required this.moveExitWidgetSelectionButtonBuilder,
   });
 
-  final Widget button;
-  final GlobalKey buttonKey;
+  final ExitWidgetSelectionButtonBuilder exitWidgetSelectionButtonBuilder;
+  final MoveExitWidgetSelectionButtonBuilder?
+      moveExitWidgetSelectionButtonBuilder;
 
   @override
-  State<_ExitWidgetSelectionButtonWrapper> createState() =>
-      _ExitWidgetSelectionButtonState();
+  State<_ExitWidgetSelectionButtonGroup> createState() =>
+      _ExitWidgetSelectionButtonGroupState();
 }
 
-class _ExitWidgetSelectionButtonState
-    extends State<_ExitWidgetSelectionButtonWrapper> {
-  bool _tooltipVisible = false;
+class _ExitWidgetSelectionButtonGroupState
+    extends State<_ExitWidgetSelectionButtonGroup> {
+  static const double _kExitWidgetSelectionButtonPadding = 4.0;
+  static const double _kExitWidgetSelectionButtonMargin = 10.0;
+
+  final GlobalKey _exitWidgetSelectionButtonKey = GlobalKey(
+    debugLabel: 'Exit Widget Selection button',
+  );
+
+  String? _tooltipMessage;
+
+  bool _leftAligned = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final Widget? moveExitWidgetSelectionButton =
+        widget.moveExitWidgetSelectionButtonBuilder != null
+            ? Padding(
+                padding: _leftAligned
+                    ? const EdgeInsets.only(
+                        left: _kExitWidgetSelectionButtonPadding,
+                      )
+                    : const EdgeInsets.only(
+                        right: _kExitWidgetSelectionButtonPadding,
+                      ),
```

Comment:
nit: this can be condensed
```
padding: const EdgeInsets.only(
  left: _leftAligned ? _kExitWidgetSelectionButtonPadding : 0.0,
  right: _leftAligned ? 0.0 : _kExitWidgetSelectionButtonPadding,
),
```

Reply:
Ooh thank you!